frontend/src/components/ProjectSettingPanel.vue

```diff
@@ -10,8 +14,10 @@
           :allow-edit="allowEdit"
         />
       </div>
+      <div v-if="isDev" class="pt-4">
```

Comment:
Looks like we don't need `isDev` here? Should this used for `ProjectBranchProtectionRulesSettingPanel` component?

Reply:
Using it on `ProjectBranchProtectionRulesSettingPanel`'s parent node to avoid rendering some useless elements with style.